geo-types/src/geometry_collection.rs

```diff
@@ -11,12 +11,33 @@ pub struct GeometryCollection<T>(pub Vec<Geometry<T>>)
 where
     T: CoordinateType;
 
+impl<T: CoordinateType> GeometryCollection<T> {
+    /// Return an empty GeometryCollection
+    pub fn empty() -> GeometryCollection<T> {
```

Comment:
```suggestion
    pub fn new() -> GeometryCollection<T> {
```

thoughts on calling this `new`? this matches other empty collection constructors in rust, e.g. `Vec::new`, `HashMap::new`

Reply:
or implement `Default` ? even if I'm not sure it makes sense for a GeometryCollection